mergify_engine/web/asgi.py

```diff
@@ -11,18 +11,11 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-from datadog import statsd
-import ddtrace
 from sentry_sdk.integrations.asgi import SentryAsgiMiddleware
 
 from mergify_engine import config
-from mergify_engine import logs
 from mergify_engine.web.root import app as application
 
 
 if config.SENTRY_URL:
     application = SentryAsgiMiddleware(application)  # type:ignore[assignment]
```

Comment:
I guess you forget `service.setup()` here ?

Reply:
yes